IotcRestApi/regulator_draft.md

```diff
@@ -131,8 +131,43 @@ Ukázka response:
 
 ```yaml
 {
-    "released-physical-device-ids": [
-        "889bfe0f-3c93-421a-9bf1-9bc26ba32ad8",
-        "2e7f00fa-ce15-47d2-97d1-6fa8a8098d37"]
+    "releasedPhysicalDeviceIds": ["abc123", "abc456", "abc789"]
+}
+```
+
+### POST api/temperature-regulator/
```

Comment:
PUT

Reply:
No to nevím, to si myslím že nebude idempotentní protože pokud jednou vytvořím device tak ho se stejnými physical devices nemůžu znovu vytvořit a obdržím chybu - něco jako že Device xy is already used in another device ... Nebo ne?

Reply:
Pokud je tam requestId ktery slouzi jako idempotency token tak to bude urcite idempotentni.

Pokud znovu vytvoris stejny device s jinym RequestId tak to opravdu hodi chybu jak sam pises ale to uz je jiny request.

Reply:
No to je pravda. Pokud pošle to stejné tak to může být jak píšeš? Ale jak zajistit že se to bude chovat správně - co když pošle stejné requestId a jiné physicalDevices?

Reply:
v tom pripade nekde udelal chybu ale z defunice toho ze requestId je idempotenci token a unikatne identifikuje request by jsme se v tu chvili meli tvarit jako ze je to stejny request.